src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -41,6 +87,7 @@ class PromptSource(Enum):
 class OutputFormat(Enum):
     OPENAI_CHAT_COMPLETIONS = auto()
     OPENAI_COMPLETIONS = auto()
+    OPENAI_VISION = auto()
```

Comment:
The response format for chat VLMs is the same as the regular chat completion since we just have text out, why have a separate entry?

Reply:
The name of the enum is a bit misleading 😅 The `OutputFormat` enum is actually not about the format of the response but it's about the format of the resulting input json file by LlmInputs.